app/views/items/_form.html.erb

```diff
@@ -21,10 +21,12 @@
       <%= f.text_field :description %>
     </p>
 
-    <p>
-      <%= f.label :quantity %><br>
-      <%= f.text_field :quantity %>
-    </p>
+    <% if is_admin? %>
+        <p>
+          <%= f.label :quantity %><br>
+          <%= f.text_field :quantity %>
+        </p>
+    <% end %>
```

Comment:
I'm not sure how a malicious user could inject this parameter somehow, bypassing this check, and if we need to check in the backend somehow as well.